src/world/Server/WorldSession.h

```diff
@@ -346,8 +346,11 @@ class SERVER_DECL WorldSession
         void handlePlayerLoginOpcode(WorldPacket& recvPacket);
         void handleCharRenameOpcode(WorldPacket& recvPacket);
         void handleCharCreateOpcode(WorldPacket& recvPacket);
-        void handleSetPlayerDeclinedNamesOpcode(WorldPacket& recvPacket); // declined names (Cyrillic client)
-        void handleCharEnumOpcode(WorldPacket& /*recvPacket*/);
+
+		// declined names (Cyrillic client)
+		void handleDeclinedPlayerNameOpcode(WorldPacket& recvPacket);   //> TBC
```

Comment:
Do not rename this function because it matches with the opcode name. Or if you want to rename the function, rename the opcode as well